packages/elastic-apm-synthtrace/src/lib/apm/service.ts

```diff
@@ -15,6 +15,7 @@ export class Service extends Entity<ApmFields> {
     return new Instance({
       ...this.fields,
       ['service.node.name']: instanceName,
+      'host.name': instanceName,
```

Comment:
This is great! Thanks for improving synthtrace also! 
@dgieselaar: looks good to you?